data/start0.json

```diff
@@ -1,9 +1,9 @@
 {
-    "platforms": [
-		{"image": "ground", "x": 0, "y": 546}
-    ],
-	"coin": [
-        {"image": "grass:6x1", "x": 390, "y": 370}
```

Comment:
This looks like a bug, it was labeled "coin" not "coins" and the coin was also a grass image?

Reply:
I see, this is the one "coin" for the start game. I can rectify this.